### PR TITLE
add command history feature

### PR DESCRIPTION
**Pull request by kevinswk94**

### CODE REVIEW

**Comment by AdarshChugani** [comment]
In future, please try to have a description.

A good description would be: `a`

**Comment by kevinswk94** [comment]
> In future, please try to have a description.
> 
> A good description would be: `a`

Thanks adarsh, very descriptive indeed!

File: src/main/java/cookbuddy/logic/CommandHistory.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package cookbuddy.logic;
+
+import java.util.LinkedList;
+import java.util.ListIterator;
```
Comment (AdarshChugani) [inline src/main/java/cookbuddy/logic/CommandHistory.java:4]: Smart use of existing resources!

**Review by AdarshChugani — APPROVED** [review]
Looks good!

**Pull request merged**